### PR TITLE
Project Force Deletion

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds content related to project deletion and the force delete behavior. The project page was also restructured to comply with the how-to layout format. Associating users to a project has been moved into its own page in the User Management section.

💻 [Preview](https://deploy-preview-1107--docs-spectrocloud.netlify.app/projects)
🎫 [PEM-801](https://spectrocloud.atlassian.net/browse/PEM-801)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1107#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by amarjeet27 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1107#pullrequestreview-1283978978" title="Approved">amarjeet27</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1107#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:52]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:55]:
```suggestion
1. Navigate to **Tenant Admin** > **Projects** and click the **Create Project** button.
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:72]:
```suggestion
You can remove projects by following these steps.
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:82]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:84]:
```suggestion
2. Switch to **Tenant Admin** scope.
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:86]:
```suggestion
3. Navigate to the left **Main Menu** and select **Projects**.
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:97]: You might want to add what a force delete is so they know when that situation might arise. Users might wonder why regular delete doesn't work and why would they have to force delete.

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:104]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:106]:
```suggestion
1. Navigate to **Tenant Admin** > **Projects** .
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:62]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:76]:
```suggestion
* Tenant admin access.
```

File: content/docs/07-projects.md
Comment (ritawatson) [inline content/docs/07-projects.md:92]: You could just say "Confirm the deletion."

File: content/docs/08-user-management/1.8-project-association.md
Comment (ritawatson) [inline content/docs/08-user-management/1.8-project-association.md:11]:
```suggestion
Associating a user or team with a specific project creates a clear distinction between who is allowed in the project and their access control permissions. By grouping resources together, you ensure that only designated members have access to and control over the project resources, preventing others from accidentally or intentionally modifying them. This improves resource accountability, reduces confusion and conflicts, and helps maintain the integrity and security of the project.
```

File: content/docs/08-user-management/1.8-project-association.md
Comment (ritawatson) [inline content/docs/08-user-management/1.8-project-association.md:13]:
```suggestion
User permissions are determined by the combination of their tenant and project roles, as well as any roles inherited from their team association. If a user is a *Tenant Admin*, they have admin permissions in all projects. A user with the *Project Viewer* role at the tenant level has *View* permissions for all projects. However, if a user or team has the *Project Viewer* role assigned to a specific project, they only have view access to that project. The extent of user permissions, either at the tenant or project level, determines the number of projects they can access.
```

Comment (ritawatson) [reply content/docs/08-user-management/1.8-project-association.md:13]: I try to avoid using possessive unless it's really needed.

File: content/docs/08-user-management/1.8-project-association.md
Comment (ritawatson) [inline content/docs/08-user-management/1.8-project-association.md:17]:
```suggestion
To associate a user or team with a project, use the following steps.
```

File: content/docs/08-user-management/1.8-project-association.md
Comment (ritawatson) [inline content/docs/08-user-management/1.8-project-association.md:21]:
```suggestion
* Tenant Admin access.
```

File: content/docs/08-user-management/1.8-project-association.md
Comment (ritawatson) [inline content/docs/08-user-management/1.8-project-association.md:29]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: content/docs/08-user-management/1.8-project-association.md
Comment (ritawatson) [inline content/docs/08-user-management/1.8-project-association.md:53]:
```suggestion
Click **Confirm** to create the project role and complete the project association process.
```

**Review by ritawatson — APPROVED** [review]
Just tightened some of the language and offered clarifications in a couple places. 

LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1107#pullrequestreview-1283978978" title="Approved">amarjeet27</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1107#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1107#pullrequestreview-1285662760" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/07-projects.md
Comment (karl-cardenas-coding) [reply content/docs/07-projects.md:23]: I didn't write this one, but I agree it needs clean-up

**Pull request merged**
